Modules/socketmodule.c

```diff
@@ -1227,6 +1227,14 @@ makesockaddr(SOCKET_T sockfd, struct sockaddr *addr, size_t addrlen, int proto)
        }
 #endif /* AF_NETLINK */
 
+#if defined(AF_VSOCK)
+       case AF_VSOCK:
+       {
+           struct sockaddr_vm *a = (struct sockaddr_vm *) addr;
+           return Py_BuildValue("II", a->svm_cid, a->svm_port);
```

Comment:
I've restored the original aclocal.m4 and fixed the issues you pointed out. 

Thank you for your review,

Cathy